networkit/cpp/graph/test/GraphGTest.cpp

```diff
@@ -147,6 +147,22 @@ TEST(GraphGTest, testDefConstructorWithDirIndex) {
     EXPECT_EQ(GDir.edgeId(0, 1), 0);
 }
 
+TEST(GraphGTest, testCopyConstructorWithIndexedEdgeIds) {
+    Graph G(3, false, false, true);
+    G.indexEdges(true);
+    G.addEdge(0, 1);
```

Comment:
Better test with a small array of edgeId (so that it is clear, that the same ids still apply after the copy).